ci-operator/config/3scale/3scale-operator/3scale-3scale-operator-3scale-2.14-stable.yaml

```diff
@@ -0,0 +1,53 @@
+binary_build_commands: make manager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.8"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: test-unit
+  commands: |
+    unset GOFLAGS
+    make test-unit
+  container:
+    from: src
+- as: test-e2e
+  steps:
+    cluster_profile: aws
+    env:
+      RELATED_IMAGE_APICAST: quay.io/3scale/apicast:latest
```

Comment:
instead of changing images.go here https://github.com/3scale/3scale-operator/pull/950/files#diff-0ee170315970baf25efa7cdc0e01f705260c330ef86b4db126b27b6def3bb43fL449

You can set the images to be tested with those env vars. Up to you.

Reply:
just take into account that env vars will override values in image.go

Reply:
Thank you for comment.  Added tag: 3scale-2.14.1-GA